### PR TITLE
Add video prop to content

### PR DESCRIPTION
**Pull request by conrad-vanl**
Adds a `video` prop to `ContentData`.
Note: built this blindly, untested.

Currently returns an `embedUrl` pointing out an `html` embeddable player. We can use this to abstract the ooyala integration, but keep all of ooyala's customizations and analytics events.

If we don't care about those - I can add a `videoUrl` prop instead and potentially point to the video resource, but I might need insight on how we can generate that link.

@Lepozepo thoughts on this?

Currently a few todos in the code... need to be addressed.

### CODE REVIEW

**Comment by newspringweb** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by conrad-vanl** [comment]
@IsaacHardy when you get an opportunity, could you look this over and let me know what you think? Basically, this gives us a spot to add video attributes on content. Right now I'm just adding an `embedUrl`, which is a url that a website or app could use to embed the player within a webview or iframe. In this case, it links to the ooyala player.

I left some comments about env vars, and a link to the existing JSON that holtzman was using to configure the player. Those env vars were actually just inside the source code on holtzman, not env vars...but I didn't know where we should put them on Heighliner for sure.  Lastly, that JSON file probably needs to be moved to an S3 endpoint, it's just important that it has CORs ability and it's served over HTTPS.

I'm working on the frontend side of this and should have a PR opened over on newspring/apollos soon.

**Comment by IsaacHardy** [comment]
Hey @conrad-vanl - I'll look this through after our morning meeting around 8:30a!

File: src/expression-engine/models/content/resolver.js
Comment (delianides) [inline src/expression-engine/models/content/resolver.js]: these are public player codes from ooyala. you can find them now as part of the video embed.

**Review by delianides — APPROVED** [review]

**Comment by IsaacHardy** [comment]
@conrad-vanl - The two variables are fine to stay as they are, because that bit of information is public knowledge. 

Regarding the JSON file, we are currently looking into a previous JSON file we already have up on S3.

File: src/expression-engine/models/content/resolver.js
Comment (delianides) [inline src/expression-engine/models/content/resolver.js]: you can use this instead
`//d3n6tjerleuu41.cloudfront.net/newspring/skin.new.json`

**Review by IsaacHardy — CHANGES_REQUESTED** [review]
Only request is the `playerConfig` change.

**Comment by conrad-vanl** [comment]
@IsaacHardy thanks for the review! I updated the player config prop to the one that @delianides  supplied.

**Comment by conrad-vanl** [comment]
Was also able to fully test this locally - should be G2G! Thanks VPN!! 🙌

**Review by IsaacHardy — APPROVED** [review]

**Comment by conrad-vanl** [comment]
@IsaacHardy thanks for the approve. FYI, we don't have push access to this repo so I can't merge myself, I think you'll have to handle any merge/deploys.

**Comment by IsaacHardy** [comment]
@conrad-vanl We are trying to determine the best means for merging in y'alls changes to master. We are trying to figure out a good way to handle simultaneous development from Differential and NewSpring. Any ideas are also welcomed!

Once we figure out a plan of action, we will proceed with the merge. 

Are y'all in dire need of these changes to be merged into the master branch?

**Comment by conrad-vanl** [comment]
> Are y'all in dire need of these changes to be merged into the master branch?

Nope! All is good for now.

> Any ideas are also welcomed!

We can talk about it on standup this week (or sooner if you'd like)

**Pull request merged**
